tests/features/consultation.feature

```diff
@@ -73,3 +97,51 @@ Feature: Consultation content creation
     And I fill in "Subject" with "export financing"
     And I press "Save"
     Then I should not see the text "The text to remove."
+
+  @javascript
+  Scenario: Test visibility of document references and ensure that document reference and contact is not deleted after removing from the node.
+    Given I am an anonymous user
+    And the following General Contact entity:
+      | Name | A general contact |
+    And the following document:
+      | name          | file       |
+      | My Document 3 | sample.pdf |
+    And the following Document "Document reference" entity:
+      | Name     | Document reference to My Document 3 |
+      | Document | My Document 3                       |
+    And the following Consultation Content entity:
+      | Title             | Consultation demo page              |
+      | Summary           | Consultation summary                |
+      | Teaser            | Consultation teaser                 |
+      | Contacts          | A general contact                   |
+      | Opening date      | 2019-02-22                          |
+      | Deadline          | 2019-03-21 18:30:00                 |
+      | Target audience   | Target audience text                |
+      | Documents         | Document reference to My Document 3 |
+    When I am visiting the "Consultation demo page" content
+    Then I should see "sample.pdf"
+
+    When the sub entity Document "Document reference" "Document reference to My Document 3" is updated as follows:
```

Comment:
Might be we don't need this test case because permissions for view operation are checked in Kernel/SubEntityAccessControlHandlerTest. But might be this example will be useful in EWCMS.